### ISSUE TITLE
Change "Node" to "Signature" (back end mostly)

### ISSUE THREAD

**Issue by dongbohu** [greenelab/adage-server#138]
For historical reasons, **"signature"** has been called **"node"** in both adage server's backend and frontend. I wonder whether it is a good idea to change all occurrences of **node** (including the database table of backend and the related djangdo management commands and the URLs) into **signature** so that it will be consistent with @tj8901nm's R package and paper.  

One concern I have now is that some users may have saved bookmarks whose URL is `host://#/node/<id>`. After this change, the url will become `hostname/#/signature/<id>` and the original one will be broken. How easy is it to redirect all URLs that include `node` to the new ones?

**Comment by dongbohu** [comment]
@rzelayafavila will take care of the `help` page.

**Comment by dongbohu** [comment]
Excluded `node` occurrences in PR #149:
* All variables and string literals that are related to API, such as **/api/v0/node/...** or **/api/v0/activity/?node=xxx...**;

* app/analyze/analysis/sampleBin.js:
```
sampleClust.orderedNodes()
```
(occurred twice in this file)

**Comment by dongbohu** [comment]
Renaming on front end is done. The renaming on back end will include:
* adage/urls.py
* adage/analyze/api.py
* adage/analyze/models.py
* adage/analyze/tests.py
* adage/analyze/management/commands/create_or_update_participation_type.py
* adage/analyze/management/commands/import_activity.py
* adage/analyze/management/commands/import_node_gene_network.py
* migration files in `adage/analyze/migrations/` (should be generated automatically by Django)

The modification of `api.py` will affect the frond end again. Any front end files in `interface/src/` directory that refers to the API such as **/api/v0/node/...** or **/api/v0/activity/?node=xxx...;** should be modified too.

**Comment by mhuyck** [comment]
An issue closely related to this came up in discussion with the Hogan lab: when viewing the list of Signatures at http://adage.greenelab.com/#/signature/search?mlmodel=1 it is a bit odd that they are all named "Node###neg" or "Node###pos" (still using the term "Node").

These particular names come from the data files we are loading. It should be a simple matter to rename those to something like "Sig###neg" and "Sig###pos" right? The labels appear in at least two data files, so we would need to be consistent and thorough when doing the update to avoid breaking any references.

Let me know if you think this should be broken out as a separate issue and I will do that.

**Comment by dongbohu** [comment]
@mhuyck: The `Nodexxx` name format exists in the following two files in `adage-server/data/` directory:
```
(1) all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt
(2) node_gene_network.txt
```
In file (1), only the first line needs to be revised.
In file (2) only the first column needs to revised.

Let me create an issue and put it into `Optional for next release` milestone. Unless you or @rzelayafavila  needs my help, I am positive that I will be able to finish it this month.

**Comment by dongbohu** [comment]
Fixed in PR #300.
